torch_geometric/utils/coalesce.py

```diff
@@ -37,6 +37,31 @@ def coalesce(
 
     :rtype: :class:`LongTensor` if :attr:`edge_attr` is :obj:`None`, else
         (:class:`LongTensor`, :obj:`Tensor` or :obj:`List[Tensor]]`)
+
+    Examples:
+
+        >>> edge_index = torch.tensor([[1, 1, 2, 3], [3, 3, 1, 2]])
```

Comment:
Can we move this to a `.. code-block:: python`block? I also feel it might be more straightforward to copy/paste if we skip the `>>>`. Might be also good to add the necessary imports.

Reply:
But we cannot display the outputs if we use a `.. code-block:: python` block. Do you think this is needed?

For the imports, I have made all the functions used from the `torch_geometric.utils` and `torch` modules only.

Reply:
I'd prefer PyTorch-style code examples like that
https://github.com/pytorch/pytorch/blob/bfc6db0a5af1acf2a4cb864c334f17bcd08fc079/torch/nn/functional.py#L4375-L4390

Reply:
Got it, I guess this works for me as well.